Sources/_InternalTestSupport/SwiftTesting+Helpers.swift

```diff
@@ -11,18 +11,30 @@
 import Basics
 import Testing
 
+// MARK: File System Helpers
+
+/// Verifies that a file exists at the specified path.
+///
+/// - Parameters:
+///   - path: The absolute path to check for file existence.
+///   - sourceLocation: The source location where the expectation is made.
```

Comment:
praise: thank you for adding documentation to these files! this makes it a lot more readable.